run.py

```diff
@@ -15,6 +18,11 @@ def main():
 
     bot = Union(config=config)
 
+    try:
+        bot.load_extension("union.core.core")
+    except Exception as e:
+        print(f"Failed to load core cog.\n{e.__class__.__name__}: {e}")
```

Comment:
On the one hand we should avoid adding special cases where possible, but in this case it's acceptable because the bot literally can't continue functioning without the core cog.